packages/gestalt/src/Button.js

```diff
@@ -18,7 +19,7 @@ type Props = {|
   accessibilityExpanded?: boolean,
   accessibilityHaspopup?: boolean,
   accessibilityLabel?: string,
-  color?: 'gray' | 'red' | 'blue' | 'transparent' | 'white',
+  color?: 'gray' | 'darkGray' | 'red' | 'blue' | 'transparent' | 'white',
```

Comment:
@jennyscript I opted for adding a new `color` instead of going for a new `selected` prop.

Reply:
That makes sense for the API. If this is a new pattern do you think there's a helpful way to add that info to the doc?

Reply:
@jennyscript Updated